DigitalOffice/Tests/TimeManagementServiceTests/Mappers/WorkTimeMapperTests.cs

```diff
@@ -0,0 +1,76 @@
+using NUnit.Framework;
+using System;
+using TimeManagementService.Database.Entities;
+using TimeManagementService.Mappers;
+using TimeManagementService.Mappers.Interfaces;
+using TimeManagementService.Models;
+
+namespace TimeManagementServiceUnitTests.Mappers
+{
+    public class WorkTimeMapperTests
+    {
+        private IMapper<CreateWorkTimeRequest, DbWorkTime> mapper;
+
+        private readonly Guid FakeProjectId = Guid.NewGuid();
+        private readonly DateTime FakeDate = new DateTime(2020, 7, 29, 0, 30, 0);
+        private readonly TimeSpan FakeTime = new TimeSpan(8, 30, 0);
+        private readonly string FakeTitle = "I was working on a very important task";
+        private readonly string FakeDescription = "I was asleep. I love sleep. I hope I get paid for this.";
+        private readonly Guid FakeWorkerUserId = Guid.NewGuid();
+
+        private CreateWorkTimeRequest FakeRequest;
+
+        [SetUp]
+        public void SetUp()
+        {
+            mapper = new WorkTimeMapper();
+
+            FakeRequest = new CreateWorkTimeRequest
+            {
+                ProjectId = FakeProjectId,
+                Date = FakeDate,
+                Time = FakeTime,
+                Title = FakeTitle,
+                Description = FakeDescription,
+                WorkerUserId = FakeWorkerUserId
+            };
+        }
+
+        [Test]
+        public void ThrowExceptionIfArgumentIsNull()
+        {
+            Assert.Throws<NullReferenceException>(() => mapper.Map(null));
+        }
+
+        [Test]
+        public void ReturnRightModelSuccesfully()
+        {
+            var result = mapper.Map(FakeRequest);
+
+            Assert.IsNotNull(result);
+            Assert.IsInstanceOf<DbWorkTime>(result);
+            Assert.AreEqual(FakeProjectId, result.ProjectId);
+            Assert.AreEqual(FakeDate, result.Date);
+            Assert.AreEqual(FakeTime, result.Time);
+            Assert.AreEqual(FakeTitle, result.Title);
+            Assert.AreEqual(FakeDescription, result.Description);
+            Assert.AreEqual(FakeWorkerUserId, result.WorkerUserId);
```

Comment:
Add SerializerAssert, serialize objects to the json objects inside methods of class SerializerAsser and further compare in one line.

Reply:
Fixed.